docs/builds/guides/development/dll-builds.md

```diff
@@ -149,6 +149,12 @@ For example:
 		} );
 ```
 
+## Live implementation sample
+
+Presented below is a working sample editor using the DLL mechanism. Observe the source and then click **"Result"** to switch to the live view of the working CKEditor 5 instance.
+
+<iframe width="100%" height="600" src="//jsfiddle.net/ckeditor/mv05bweu/8/embedded/html,result/dark/" allowfullscreen="allowfullscreen" allowpaymentrequest frameborder="0"></iframe>
```

Comment:
When we set the final content of the sample, we should set a current version as the base.

![image](https://user-images.githubusercontent.com/2270764/131493380-92d82877-7c59-42b1-8aa3-f2fcdb5b0e59.png)

Then, the URL will not contain the `/8/` part.

Reply:
Done.